action/README.md

```diff
@@ -7,19 +7,63 @@ Requires Go to be installed prior to running (e.g. actions/setup-go).
 
 ## Inputs
 
-### `cmd`
+<!--BEGIN GENERATED DOCS-->
+
+### `outputFile`
+
+The output file from the function. Default value: `'function_output.json'`.
```

Comment:
Could we somehow separate the input description from the input default value if we're generating docs?